src/gcuserview.cpp

```diff
@@ -156,10 +157,10 @@ class GCUserViewDelegate : public QItemDelegate
             if(ava.isNull()) {
                 ava = IconsetFactory::iconPixmap("psi/default_avatar");
             }
-            ava = AvatarFactory::roundedAvatar(ava, avatarRadius_, avatarSize_);
+            ava = AvatarFactory::roundedAvatar(ava, avatarRadius_ * devicePixelRatio(list_), avatarSize_ * devicePixelRatio(list_));
```

Comment:
You don't need all these changes to this file. I just commited a change similar to regular roster.